CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ Please keep one empty line before and after all headers. (This is required for `
 
 And please only add new entries to the top of this list, right below the `# Unreleased` header.
 
-# Unreleased
+# 0.27.0
```

Comment:
Please keep the `# Unreleased` header

Reply:
I don't think that this date makes any sense here. The date would be on github release anyway and on the tag. Having date here doesn't provide any value at all. And if someone really want that date they can check blame as well. it's just nearly impossible to keep this date right, also it's not accurate due to timezones, etc, etc.

Removing unreleased is also expected and widely used.

Reply:
> Removing unreleased is also expected and widely used.

That only seems annoying / cause extra conflicts for the next contributor?

Reply:
I think it's fine to re-add the `# Unreleased` header in the next PR to be merged. It's not a difficult change, and we're usually given permission to push to the PR author's branch, so we're not necessarily going to have to wait on the PR author to resolve that.

As for the date: I don't think it's a problem that it can be off by a day or so. Maybe having a placeholder like `2022-XX-XX` before it's clear when the update will happen could be "good enough". I do have to agree that it doesn't add much value, but I consider it a "nice to have" sort of thing.

Reply:
> we're usually given permission to push to the PR author's branch, so we're not necessarily going to have to wait on the PR author to resolve that.

Right, this was more of a problem when we didn't do that.

Re dates: I meant this review most to remember to actually include the date. I think it's nice to have dates in changelogs, but will gladly discuss in another issue if we should stop doing so (though I think we should remove them from the rest of the changelog as well if so).